.gitignore

```diff
@@ -1,5 +1,30 @@
-node_modules
-web-ext-artifacts
+# Numerous always-ignore extensions
+*.diff
+*.err
+*.orig
+*.log
+*.rej
+*.swo
+*.swp
+*.vi
+*~
+
+# OS or Editor folders
 .DS_Store
-src/libs/*
-!.gitkeep
```

Comment:
this is a rather sensible default

Reply:
You don't have to create a negative pattern for `gitkeep` to make it save to the git repository. Look for an example at the repo with some gitignore templates: https://github.com/github/gitignore/blob/main/Node.gitignore